README.md

```diff
@@ -34,14 +34,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+    Responsive rearranges the design using CSS to fit different size screens. Adaptive has a specific design that loads for different size screens.
+
```

Comment:
Also, responsive designs mainly use percentages to allow the view to fit any size, while adaptive will work with breakpoints at certain sizes.